StortSpelProjekt/Sandbox/src/main.cpp

```diff
@@ -58,6 +72,14 @@ int WINAPI WinMain(HINSTANCE hInstance, HINSTANCE, LPSTR, int nCmdShow)
         /* ------ Sort ------ */
         renderer->SortObjects();
 
+        /* ------ ImGui ------*/
+        if (DEVELOPERMODE_DEVINTERFACE == true)
+        {
+            ImGuiHandler::GetInstance().UpdateFrame();
+        }
```

Comment:
Same here, can this be put inside Renderer? Since this is a "developerMode" config, maybe it shouldn't be visible in "main"?